src/Microsoft.AspNetCore.Mvc.Core/Internal/ControllerArgumentBinder.cs

```diff
@@ -241,7 +289,8 @@ private static void CallPropertyAddRange<TElement>(object target, object source)
 
         private async Task<IDictionary<string, object>> PopulateArgumentsAsync(
             ControllerContext controllerContext,
-            IList<ParameterDescriptor> parameters)
+            IList<ParameterDescriptor> parameters,
+            IValueProvider valueProvider)
```

Comment:
Missing something here: The new parameter isn't used yet and this method still calls the older `BindModelAsync()` overload.